apps/etterlatte-saksbehandling-ui/client/src/components/behandling/trygdetid/Trygdetid.tsx

```diff
@@ -71,6 +74,22 @@ export const Trygdetid = ({ redigerbar, behandling, virkningstidspunktEtterNyReg
     )
   }
 
+  const oppdaterYrkesskade = (yrkesskade: Boolean) => {
+    if (!trygdetid?.id) throw new Error('Mangler trygdetidid')
+    if (!behandling?.id) throw new Error('Mangler behandlingsid')
```

Comment:
Gjør disse feilene synlig for brukerene på en måte

Reply:
Den var tatt fra den eksisterende kode lenger ned i fil:

```
  useEffect(() => {
    if (!behandling?.id) throw new Error('Mangler behandlingsid')

    fetchTrygdetid(behandling.id, (trygdetid: ITrygdetid) => {
 
 ...
 ```
 
 Man burde ikke kommet så langt som denne siden uten verdiene. Mistenker at det var satt opp slik for å force ? til ! :)
 
 Men - må vel fikse begge steder - både gammel og ny